### PR TITLE
use pre-built kubeadm box to speed up install

### PR DESCRIPTION
**Pull request by rchirakk**
use pre-built kubeadm box to speed up install

### CODE REVIEW

File: cluster/Vagrantfile [hunk 1]
```diff
@@ -102,7 +102,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       elsif node_os == UBUNTU
         c.vm.box = 'ubuntu/xenial64'
       else
-        c.vm.box = 'centos/7'
+        if ENV['CONTIV_KUBEADM']
+           c.vm.box = 'contiv/kubeadm-centos'
```
Comment (unclejack) [inline cluster/Vagrantfile]: @rchirakk: How was this box built? Do we have some scripts/documentation on how to build this VM?

Comment (rchirakk) [inline cluster/Vagrantfile]: I have packaged the vagrantfile in the box itself for reference/future changes.

Comment (neelimamukiri) [inline cluster/Vagrantfile]: @rchirakk, how much time does this save? What is the process to update the kubeadm version or k8s version?

Right now the k8s version is configurable and updating it is very simple. Most users using kubeadm also go the the new versions very soon.

Comment (rchirakk) [inline cluster/Vagrantfile]: it saves around 10 mnts . You can update kubeadm/k8s version during install or in the box image

Comment (neelimamukiri) [inline cluster/Vagrantfile]: Sounds okay to me. Please add the instructions to add the box in the contiv/build repo, so this is repeatable.

Comment (rchirakk) [inline cluster/Vagrantfile]: it is included in the image

**Pull request closed without merging**